packages/terra-modal/src/modal.scss

```diff
@@ -0,0 +1,79 @@
+.terra-Modal-backdrop {
+  background-color: #1c1f21;
+  border: 0;
+  height: 100%;
+  left: 0;
+  opacity: 0.8;
+  padding: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1000;
```

Comment:
will update this when this pull request is merged: https://github.com/cerner/terra-legacy-theme/pull/12.
or if the <Overlay /> component is completed, we can swap the backdrop with overlay